src/Pages/instructor/manage-class/manage-class.js

```diff
@@ -3,19 +3,150 @@ import styled from 'styled-components';
 
 import { StyledLink, MainHeader, Banner } from "../../../reusable-components/reusableComponents";
```

Comment:
Comment these out if you're not using any of them just to get rid of Linter warnings in terminal